src/collection-browser.ts

```diff
@@ -658,6 +667,11 @@ export class CollectionBrowser
     `;
   }
 
+  private toggleManageViewRemoveButtonState() {
+    this.manageViewRemoveButtonEnable =
+      !!this.dataSource.checkedTileModels.length;
```

Comment:
Suggestion: When I use `!!` I get a linting error. If I'm correct, TS style suggests using the same type for comparisons